MedTracker/Controllers/DoctorController.cs

```diff
@@ -139,7 +139,6 @@ public async Task<IActionResult> GetPatients()
                 .Select(t => new
```

Comment:
You can combine this and the query bellow using `Join`s.

Reply:
I dont understand this problem

Reply:
Clean code perspective.
Just look at the first query. You are selecting `IdTreatment` and `DoctorID`, but you are not even using `DoctorID` further in code. The line 146 could be removed, if you selected from the start just `ItTreatment`. Second, the code becomes smaller with just one query. More lines of code, means more lines to read and to understand (in our case, not every time)

Reply:
Now it is a single query